ui/src/ScholarReader.tsx

```diff
@@ -203,16 +203,30 @@ class ScholarReader extends React.Component<ScholarReaderProps, State> {
     this.setState({ userAnnotations: annotations });
   }
 
+  closeDrawerOnEscape = (event: KeyboardEvent) => {
+    if (event.key === "Escape") {
```

Comment:
There's a new helper for this! Check this out: https://github.com/allenai/scholar-reader/blob/ec635d0e07b8c04565d2e65cbfbe68b8ba852705/ui/src/SelectionLayer.tsx#L58

The level of carefulness of checking for the Escape key may, however, be unnecessary.

Reply:
Awesome!

I'll update.

Reply:
Ah, interesting. The utility expects a `React.KeyboardEvent`, but in this case we're handling a native, `KeyboardEvent`.

I wonder if I can somehow create a `React.KeyboardEvent` from a raw `KeyboardEvent`, or overload the method to accept either.

Reply:
FYI, here's the change:
c4e4d984bc2cab52a1b4eed423a0386503e5aea4

Reply:
Oh, I overlooked that! Either approach sounds fine to me. I could also be persuaded that the isKeypressEscape utility is unnecessarily complex.

Reply:
I was late to the party with my last comment XD Looks good!